environment.cpp

```diff
@@ -69,12 +69,6 @@ double calculate_optimal(const environment &e, std::vector<double> input)
   return e.get_current_function()(input);
 }
 
-//void environment::update_optimal()
-//{
-//  m_optimal_output = calculate_optimal();
-//}
```

Comment:
What happened? you just removed this?

Reply:
Yeah, it was a thing I had commented long ago and didn't need anymore (because now optimal is in simulation) and I came across it while checking the code for everything I had commented out, so I just deleted it